generative_data_prep/__main__.py

```diff
@@ -302,12 +304,40 @@ def get_categories(categories_path: str):
     return category_to_id
 
 
+def process_input_dir(input_path: str) -> str:
+    """Processes a directory containing JSONL files and combines them into a single output file.
+
+    Args:
+        input_path (str): The path to the directory containing the input JSONL files.
+
+    Returns:
+        str: The path to the combined output JSONL file. If there is only one JSONL file
+             in the directory, returns the path to that file directly without combining.
+    """
+    input_jsonl_files = glob(f"{input_path}/*.jsonl")
```

Comment:
Do we need to raise an error if no files are found here ?

Reply:
Done